deploy/config/rbac/cluster_role.yaml

```diff
@@ -65,6 +65,9 @@ rules:
 - apiGroups: ["addon.open-cluster-management.io"]
   resources: ["clustermanagementaddons/finalizers"]
   verbs: ["update"]
+- apiGroups: ["addon.open-cluster-management.io"]
+  resources: ["clustermanagementaddons/status"]
+  verbs: ["patch", "update"]
```

Comment:
Do we also have to update `deploy/olm-catalog/manifests/submariner-addon.clusterserviceversion.yaml`

Reply:
Indeed, thanks @sridhargaddam!